_pages/cfw.md

```diff
@@ -1,6 +1,45 @@
 ---
 layout: page
 title: "BSidesSF 2022 Call For Workshops"
---- 
+permalink: /cfw.html
+---
+
+**What:** BSidesSF 2022 Call For Workshops
+
+**When:** BSidesSF will take place in San Francisco from February 5-7, 2022. The Call for Participation for talks and workshops will close on October 11, 2021 at 11:59 PM PST.
+
+**How:** Complete the Call for Papers Form
+
+**Length:** Short (2 hours, 45 minutes) and Long (5 hours, 30 minutes)
+
+-**Important Dates**
+- August 30, 2021 – Call for submissions opens
+- September 20, 2021 – Workshop proposals for early-bird consideration due
+- October 11, 2021 – Final due date for all submissions, rolling notifications begin
+- November 15, 2021 – All notifications, including waitlist, sent
+- February 5, 2022 - Workshops Presented
+
+**About BSidesSF**
+
+BSidesSF is a non-profit organization designed to advance the body of Information Security knowledge, by providing an annual, two day, open forum for discussion and debate for security engineers and their affiliates. We produce a conference that is a source of education, collaboration, and continued conversation for information technologists and those associated with this field. The technical and academic presentations at BSidesSF are given in the spirit of peer review and advanced knowledge dissemination. This allows the field of Information Security to grow in breadth and depth, and continue in its pursuit of highly advanced scientifically based knowledge.
+
+The BSides community has continuously raised the bar and put the INFO back in INFOSEC. We thank each and every member/participant/organizer of this community for their hard work, sweat, and relentless pursuit of High Quality information.
+
+With some of the most passionate and influential infosec practitioners from around the country, and the world, coming to RSA Conference, BSidesSF is the place to be. The mix ranges from “down in the trenches” engineers, to business leaders, thought leaders, and executive decision makers of all stripes.
+
+# What You Need
+- Trainer name(s) and contact information
+- Bio(s)
+- Course Title
+- Category / Type of Training
+- Maximum # of students that can be accomodated (limit 40)
+- Full abstract describing the course, any prerequisites, special requirements, etc.
+- Detailed outline of the entire course.
+- Description of any course materials, equipment, etc.
+- List of conferences at which course has been presented or submitted.
+- Why do you feel this submission is a good fit for BSidesSF?
+- Have you presented other courses at any other conferences? If yes, which ones?
+
+Sound good? Complete the Call for Workshops Form [here!](https://docs.google.com/forms/d/e/1FAIpQLSeb2pDvmLfrg4oJ6dGbd1I_wIeKTnIcqqCvuZVGnfyeDQnF-g/viewform)
```

Comment:
Any reason we're not using Papercall for this? The form needs a few edits. I'll poke on Slack.